src/utils/logging.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export interface LogInterface {
+    debug(msg: string, ...supportingData: any[]): void;
+    warn(msg: string, ...supportingData: any[]): void;
+    error(msg: string, ...supportingData: any[]): void;
+    info(msg: string, ...supportingData: any[]): void;
+}
```

Comment:
Do we really need the supportingData? I prefer to avoid any types, unless absolutely needed.

Reply:
Done, removed